src/main/java/com/ksm/wstbackoffice/exception/ErrorResponseMessage.java

```diff
@@ -0,0 +1,15 @@
+package com.ksm.wstbackoffice.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class ErrorResponseMessage {
+    private int statusCode;
+    private Date timestamp;
```

Comment:
You call it a timestamp but return just Date?
Or Date works like date + time?

Reply:
Not answered

Reply:
changed know return -> "localDate": "2021-01-03",